JASP-R-Interface/jaspResults/src/jaspModuleRegistration.h

```diff
@@ -30,6 +30,7 @@ RCPP_MODULE(jaspResults)
 		.property("title",							&jaspObject_Interface::getTitle,	&jaspObject_Interface::setTitle,	"Set the title of this object")
 		.property("warning",						&jaspObject_Interface::getWarning,	&jaspObject_Interface::setWarning,	"Set a warning on this object to be shown in the output")
 		.property("position",						&jaspObject_Interface::getPosition,	&jaspObject_Interface::setPosition,	"Set the position of this object in it's container. By default this is at the end in the order of adding. You can specify any other value, they do not need to be next to each other or unique. The rule is: lower values (including negative) are higher in the container and when multiple objects in a container have the same position-value order is derived from adding-order.")
+  		.property("error",                          &jaspObject_Interface::getError,	&jaspObject_Interface::setError,	"Set an error on this object. Will make the output in JASP greyed out. Errors set on jaspContainers or jaspResults are propagated to children.")
```

Comment:
In response to https://github.com/jasp-stats/jasp-desktop/pull/3032#discussion_r255084333

Just remove `,	&jaspObject_Interface::setError` here and then the property will be read-only.
Then setError can be added as a method.

While I agree that this is perhaps contrary to what an R-user would expect I think this is the best option. Because how confusing would it be if you set error to false and it remains true anyway?